src/pages/Product.js

```diff
@@ -4,6 +4,7 @@ import { MainNav, Footer } from "../components";
 import { getProductoByIdAsync } from "../actions/productosActions";
 import { isAuthenticated } from "../utils/isAuthenticated";
 import { PUBLIC_URL } from "../config/constants";
+import "../colormodificado.css";
```

Comment:
Please use english for the name of the file, for example

`Product.module.css`